lk2nd/device/dts/msm8996/msm8996-xiaomi-mi5.dts

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8996 0x30001>;
+	qcom,board-id = <31 0>, <32 0>;
+	qcom,pmic-id = <0x20009 0x2000a 0x00 0x00>;
```

Comment:
Does `pmic-id` necessary?

Reply:
i think so,

Reply:
I think it could boot with out it but nvm it was already merged